NEWS.md

```diff
@@ -1,4 +1,21 @@
+# 1.0.0 (2020-04-xx)
```

Comment:
Let's keep under `0.y.z`  as they allow breaking changes as well and we might don't want to make them 1.0 FTM.